.github/workflows/ci.yml

```diff
@@ -103,16 +103,16 @@ jobs:
           args: --check runtime/
 
       - if: "!cancelled()"
-        name: lintlua
-        run: make lintlua
+        name: luacheck
+        run: ctest --test-dir ${BUILD_DIR} -R luacheck
```

Comment:
Just `make lintlua` should work here, right?

Reply:
It does indeed work. 

I was hoping to find a github action that can interact with ctest directly, but we can skip that for now.